Frameworks/Foundation/NSNull.mm

```diff
@@ -14,9 +14,7 @@
 //
 //******************************************************************************
 
-#include "Starboard.h"
-#include "StubReturn.h"
-#include "Foundation/NSNull.h"
+#import <Foundation/NSNull.h>
 
 static NSNull* nullSingleton;
```

Comment:
This won't be switched to StrongId because `NSNull` cannot be retained or released, and making that switch would confer no benefit.